pkg/state/conformance/state.go

```diff
@@ -453,6 +454,18 @@ func (suite *StateSuite) testWatchKindWithLabels(useAggregated bool) {
 		state.WatchWithLabelQuery(resource.LabelExists(labelCommon)),
 	))
 
+	// watch with label == label2 || label == label3
+	suite.Require().NoError(watchAggregateAdapter(
```

Comment:
so watch returns if only `label == label3` is only matched?

Reply:
hmm, not sure I've got the question :)
But the idea is that if you provide two label query options and either of them matches we add the item to the watch.

So the list will contain both res with `label = label2` and `label = label3`